Sources/Components/Broadcast/BroadcastContainerView/BroadcastContainerView.swift

```diff
@@ -0,0 +1,258 @@
+
+    /// Called when the BroadcastContainerView is about to dismiss a broadcast
+    ///
+    /// - Parameters:
+    ///   - broadcastContainerView: the BroadcastContainerView
+    ///   - index: the index of the broadcast that is about to be dismissed
+    ///   - newContainerSize: the size that the BroadcastContainerView will have when the broadcast is dismissed
+    ///   - commitToDismissal: a closure that must be called in order for the BroadcastContainerView to dismiss the broadcast
+    func broadcastContainer(_ broadcastContainerView: BroadcastContainerView, willDismissBroadcastAtIndex index: Int, withNewContainerSize newContainerSize: CGSize, commitToDismissal: @escaping (() -> Void))
+}
+
+public protocol BroadcastContainerViewDataSource: class {
+    /// The number of broadcasts to display in the BroadcastContainerView
+    ///
+    /// - Parameter broadcastContainerView: the BroadcastContainerView
+    /// - Returns: The number of broadcasts to display in the BroadcastContainerView
+    func numberOfBroadcastsToDisplay(in broadcastContainerView: BroadcastContainerView) -> Int
+
+    /// The broadcast to display at the index
+    ///
+    /// - Parameters:
+    ///   - broadcastContainerView: the BroadcastContainerView
+    ///   - index: the index for the broadcast
+    /// - Returns: The broadcast to display at the index
+    func broadcastContainerView(_ broadcastContainerView: BroadcastContainerView, broadcastForIndex index: Int) -> Broadcast
+}
+
+/// A view that is used to display multiple BroadcastViews
+public final class BroadcastContainerView: UIView {
+    private struct LayoutConstants {
+        struct ContentView {
+            static let insets = Insets(top: .mediumLargeSpacing, leading: .mediumLargeSpacing, bottom: 0, trailing: .mediumLargeSpacing)
+        }
+
+        struct BroadcastView {
+            static let spacing: CGFloat = .mediumLargeSpacing
+        }
+    }
+
+    /// The datasource for the BroadcastContainerView
+    public weak var dataSource: BroadcastContainerViewDataSource? {
+        didSet {
+            reload()
+        }
+    }
+
+    /// The delegate of the BroadcastContainerView
+    public weak var delegate: BroadcastContainerViewDelegate?
+
+    private lazy var contentView: UIStackView = {
+        let view = UIStackView(frame: .zero)
+
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.axis = .vertical
+        view.spacing = LayoutConstants.BroadcastView.spacing
+        view.distribution = .fillProportionally
+
+        return view
+    }()
+
+    public override init(frame: CGRect) {
+        super.init(frame: frame)
+        setup()
+    }
+
+    public required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        setup()
+    }
+
+    public override func didMoveToSuperview() {
+        contentView.frame = CGRect(x: 0, y: 0, width: frame.width, height: 0)
+    }
+}
+
+// MARK: - Public
+
+public extension BroadcastContainerView {
```

Comment:
I like the fact that you are using a public and private extensive to group methods 👌I think if we are not doing that already we should :D. Not a big fan of the `// MARK: -` in general, though, I like them better in Objective-C but in Swift they just feel like comments and I always try to make code that needs comments... We should talk about this with the rest of the team.